.github/ISSUE_TEMPLATE/bug.yaml

```diff
@@ -0,0 +1,93 @@
+name: "🐛 Bug Report"
+description: "Submit a bug report to help us improve"
+title: "🐛 Bug Report: "
+labels: [bug]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking the time to fill out our bug report form 🙏
+  - type: textarea
+    id: steps-to-reproduce
+    validations:
+      required: true
+    attributes:
+      label: "👟 Reproduction steps"
+      description: "How do you trigger this bug? Please walk us through it step by step."
+      placeholder: "When I ..."
+  - type: textarea
+    id: expected-behavior
+    validations:
+      required: true
+    attributes:
+      label: "👍 Expected behavior"
+      description: "What did you think would happen?"
+      placeholder: "It should ..."
+  - type: textarea
+    id: actual-behavior
+    validations:
+      required: true
+    attributes:
+      label: "👎 Actual Behavior"
+      description: "What did actually happen? Add screenshots, if applicable."
+      placeholder: "It actually ..."
+  - type: dropdown
+    id: utopia-version
+    attributes:
+      label: "🎲 Utopia Framework version"
+      description: "What version of Utopia are you running?"
+      options:
+        - Version 0.18.x
+        - Version 0.17.x
+        - Version 0.16.x
+        - Version 0.15.x
+        - Version 0.14.x
+        - Different version (specify in environment)
+    validations:
+      required: true
+  - type: dropdown
+    id: php-version
+    attributes:
+      label: "💻 PHP Version"
+      description: "What version of PHP are you running?"
+      options:
+        - PHP 7.3.x
+        - PHP 7.4.x
+        - Different version (specify in environment)
+    validations:
+      required: true
+  - type: dropdown
+    id: operating-system
+    attributes:
+      label: "💻 Operating system"
+      description: "What OS is your server / device running on?"
+      options:
+        - Linux
+        - MacOS
+        - Windows
+        - Something else
+    validations:
+      required: true
+  - type: textarea
+    id: environment
+    validations:
+      required: false
+    attributes:
+      label: "🧱 Your Environment"
+      description: "Is your environment customized in any way?"
+      placeholder: "I use Cloudflare for ..."
+  - type: checkboxes
+    id: no-duplicate-issues
+    attributes:
+      label: "👀 Have you spent some time to check if this issue has been raised before?"
+      description: "Have you Googled for a similar issue or checked our older issues for a similar bug?"
+      options:
+        - label: "I checked and didn't find similar issue"
+          required: true
+  - type: checkboxes
+    id: read-code-of-conduct
+    attributes:
```

Comment:
Let's make this field consistent with other repositories. Check https://github.com/utopia-php/framework/pull/28 for rference.

Reply:
@lohanidamodar I have updated the PR with suitable changes as requested in suggestions. Please review it.